CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## [2.28.0] (2023-10-04)
+
+ENHANCEMENTS:
+
+* Enhanced validation ([#1368](https://github.com/hashicorp/terraform-ls/pull/1368))
+  * Additional diagnostics for [selected invalid Terraform language constructs](https://github.com/hashicorp/terraform-ls/blob/v0.32.0/docs/validation.md#enhanced-validation) based on detected Terraform version and provider versions are provided.
```

Comment:
```suggestion
* Enhanced validation provides additional diagnostics for [selected invalid Terraform language constructs](https://github.com/hashicorp/terraform-ls/blob/v0.32.0/docs/validation.md#enhanced-validation) based on detected Terraform version and provider versions are provided.
```

I think the previous wording buries the lead by putting the description in an indented bullet point

Reply:
Yep, I agree this is better. I'd still include the link to the PR near the end though?